manifests/nodejs.yml

```diff
@@ -557,11 +558,11 @@ tests/:
       TestAdmisionControllerProfiling: *ref_5_22_0
   parametric/:
     test_config_consistency.py:
-      Test_Config_RateLimit: missing_feature
-      Test_Config_TraceAgentURL: *ref_5_22_0
-      Test_Config_TraceEnabled: missing_feature
+      Test_Config_RateLimit: *ref_5_25_0
+      Test_Config_TraceAgentURL: *ref_5_25_0
+      Test_Config_TraceEnabled: *ref_3_2_0
```

Comment:
did you try running the tests for all those "old" versions ?
I can see `3.2.0`, `3.13.0`, and `3.0.0` being added.
(ask me if you need help on how to do that)

Reply:
Yes, the tests are currently passing in the CI. I looked into the earliest implementation of those tests to verify their success. The remaining tests required some modifications and are pending the next release, once the PR implementing those changes in dd-trace-js get released.

Reply:
Unless you mean manually running them against those versions locally?

Reply:
Yeah no I meant running those tests (locally or not) with old versions of the library to make sure the version you're putting here is correct.
The system tests CI by default will only test on the latest version released and on the master branch. It won't test old versions for you.

>  The remaining tests required some modifications and are pending the next release, & so are scheduled to be run once the PR implementing those changes in dd-trace-js get released.

If your PR is in master, it should already be tested in this CI, see the `Testing the test / System Tests (nodejs, dev)` jobs. "dev" -> master, "prod" -> latest release.

Reply:
Ok updated manifest file after testing those versions locally